### DISCUSSION TITLE
Using matrix as a IRC (slack-like) channel?

### DISCUSSION THREAD

**Discussion by rocky** [Mathics3/mathics-core#610] [General]
On the failing mathics slack channel David Melik suggested switching slack to [Matrix](https://meta.wikimedia.org/wiki/Matrix.org).

Thoughts about this? 

I have created a private chat room Mathics3. If you have an account and would like an invite let me know.

**Comment by axkr** [comment]
We are using Discord for that purpose.

It's easy to create bots to add some "extra support" for Mathics evaluations.
Unfortunately (AFAIK)  the bots aren't allowed to use JavaScript in the output.
So graphs have to be images.

* https://discord.com/invite/tYknzr2qam

and the API wrapper for Discord written in Python:
* https://github.com/Rapptz/discord.py

**Reply by dchmelik** [reply]
> We are using Discord for that purpose.

Discord is a proprietary--centralized--single point of potential failure, and they don't like you using alternative chat clients.  Matrix is Free/Libre/Opensource Software (FLS, OSS, FOSS, FLOSS) and a decentralized open protocol with more chat clients, so is better--and (unlike discord) no ads!

> It's easy to create bots to add some "extra support" for Mathics evaluations. Unfortunately (AFAIK) the bots aren't allowed to use JavaScript in the output.

You can use bots in matrix also, which as far as I know would have no such restrictions.
